### PR TITLE
Fix container name property

### PR DESCRIPTION
**Pull request by cryobry**
Get container name from list of Names

### CODE REVIEW

**Comment by rhatdan** [comment]
@mwhahaha @jwhonce PTAL

**Review by mwhahaha — APPROVED** [review]
/lgtm

File: podman/domain/containers.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Container(PodmanResource):
     def name(self):
         """str: Returns container's name."""
         with suppress(KeyError):
-            return self.attrs["Name"].lstrip("/")
+            return self.attrs["Names"][0].lstrip("/")
```
Comment (mwhahaha) [inline podman/domain/containers.py:28]: The ci is failing trying to check the name property of a container. I wonder if we need to handle both cases

e.g.

 if 'Name' in attrs:
    return self.attrs["Name"].lstrip("/")
else:
    return self.aters["Names"][0].lstrip("/")

**Review by mwhahaha — CHANGES_REQUESTED** [review]
Actually per the CI it looks like this is not correct.  Maybe we need to handle both cases depending on version?

**Pull request closed without merging**